bentoml/_internal/frameworks/sklearn.py

```diff
@@ -149,7 +149,14 @@ def save_model(
     )
 
     if signatures is None:
-        signatures = {"predict": {"batchable": False}}
+        signatures = {
```

Comment:
revert this change? I think it's nicer to print a short info log that shows the default